src/cantools/database/can/message.py

```diff
@@ -453,6 +467,22 @@ def cycle_time(self) -> Optional[int]:
 
         return self._cycle_time
 
+    @property
+    def delay_time(self) -> Optional[int]:
+        """The message delay time, or ``None`` if unavailable.
+
+        """
```

Comment:
Clarify docstring

Reply:
Not quite sure, is this the case? (still, what would be the difference w.r.t. `start_delay_time`)
```python
        """The offset with respect to the message period. E.g.:
            - period -> 5 ms
            - delay -> 1 ms
            - scheduled sends -> n * period + delay (6 ms, 11 ms, 16 ms, ...)
        
        ``None`` if unavailable.
        """
```
or is this the case? (still, what would be the difference w.r.t. `cycle_time`?)
```python
        """The offset with respect to the message period. E.g.:
            - period -> 5 ms
            - delay -> 1 ms
            - scheduled sends -> n * (period + delay) (6 ms, 12 ms, 18 ms, ...)
        
        ``None`` if unavailable.
        """
```
I know it seems quite picky and not prioritary ATM, but I'm afraid this ambiguity could easily lead to code using these attributes not properly.